### PR TITLE
EDU-570 Require public storage upload liability confirmation

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-570

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,15 +83,16 @@ educandu({
   }],
   additionalControllers: [],
   consentCookieNamePrefixPrefix: 'APP_CONSENT_COOKIE_NAME'
+  uploadLiabilityCookieName: 'APP_UPLOAD_LIABIKITY_COOKIE_NAME'
```
[the remark this answers is not in the document]
Comment (adrianaluchian) [reply README.md:86]: 😂

[the remark this answers is not in the document]
Comment (adrianaluchian) [reply README.md]: Kitty has retired

File: src/components/storage-browser.js [hunk 2]
```diff
@@ -732,6 +747,8 @@ export default function StorageBrowserWrapper({ ...props }) {
       user={user}
       storage={storage}
       setStorage={setStorage}
+      getUploadLiabilityCookie={() => getCookie(uploadLiabilityCookieName)}
+      setUploadLiabilityCookie={() => setLongLastingCookie(uploadLiabilityCookieName, 'true')}
```
Comment (ahelmberger) [inline src/components/storage-browser.js:751]: I think this should be `setSessionCookie` if we want it to reappear after the browser is re-opened?

**Comment by ahelmberger** [comment]
Looks dangerous enough now. Probably no-one will ever upload anything anymore. The tests are red in this PR, though

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**